docker/configure_workers_and_start.py

```diff
@@ -107,6 +107,33 @@
         "shared_extra_conf": {},
         "worker_extra_conf": "",
     },
+    "client_reader": {
```

Comment:
Complement didn't seem to have this, while sytest does...not sure what to think of that, but I wasn't sure where else to put this endpoint?

Reply:
Also I *think* this is all that is needed to add a new worker type, but I'm not 100% sure on that?